src/components/TheFooter.vue

```diff
@@ -40,12 +40,22 @@
         </div>
         <div class="footer__column">
           <div class="footer__network">
-            <div class="footer__version">
+            <app-link
```

Comment:
In my opinion I would just add the link to the version tag and not the whole text, looks a bit better to me.

Reply:
This won't work on on-production. Is that ok?

Reply:
Why wouldn't it work? It's just about to not add the link to the service name. I don't get the difference

Reply:
Sorry, minsunderstading. Of course it will :) I'll do that